.infrastructure/helm-chart/todoapp/charts/mysql/values.yaml

```diff
@@ -0,0 +1,34 @@
+mysql:
+
+  namespace: mysql
+
+  secrets:
+    data:
+      MYSQL_ROOT_PASSWORD: "MTIzNA==" # Base64 encoding for "1234"
+      MYSQL_USER: "YXBwX3VzZXI=" # Base64 encoding for "app_user"
+      MYSQL_PASSWORD: "MTIzNA==" # Base64 encoding for "1234"
+
+  image:
+    name: mysql
+    repository: mysql
+    version: "8.0"
+
+  replicas: 2
+
+  resources:
+    requests:
+      cpu: 500m
+      memory: 1Gi
+
+  volumeClaimTemplates:
+    resources:
+      requests:
+        storage: 2Gi
+
+  affinity:
+    podAffinity:
+      key: "app"
+      value: "mysql"
+    nodeAffinity:
+      key: "app"
+      value: "mysql"
```

Comment:
Fix EoF

Reply:
fixed in [fix EoF, add serviceAccount YAML file, use pv storage value from value.yml](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/pull/30/commits/72ec2793e112758cfac44d2779c6d423be281e55) commit